pkg/reconciler/v1alpha1/build/build.go

```diff
@@ -86,25 +88,40 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 		return nil
 	}
 
-	pod, err := c.reconcileBuildPod(build)
-	if err != nil {
-		return err
-	}
+	if build.Rebasable() {
+		//todo how to handle errors
```

Comment:
What does this mean?

Reply:
great question. I think we want to figure out if we want to try to rebase again in case it fails, as the reconciler will try to perform the operation again. Do we want to skip the rebase or not is the question in this situation